mixins/_buttons.pcss

```diff
@@ -3,7 +3,6 @@
  * ----------------------------------------------------------------------------- */
 
 @define-mixin button-global {
-	background-color: transparent;
```

Comment:
I think this one was added when we worked on the theme compatibility stuff. Have you checked the changes with different themes?

Reply:
Yup, I just moved the background color to base, since there are cases where we use button and not use the classes that use these mixins. I've overridden them with the `tribe-common-c-button` classes.